ErpNet.FP.Core/Drivers/BgIslFiscalPrinter.cs

```diff
@@ -253,6 +253,19 @@ public virtual (ReceiptInfo, DeviceStatus) PrintReceiptBody(Receipt receipt)
             }
             receiptInfo.ReceiptNumber = lastDocumentNumberResponse;
 
+            if (DeviceInfo.SupportPaymentTerminal && DeviceInfo.UsePaymentTerminal && deviceStatus.Ok && receipt.Payments != null)
+            {
+                var payWithCard = false;
+                foreach (var p in receipt.Payments)
+                {
+                    if (p.PaymentType == PaymentType.Card)
+                        payWithCard = true;
+                }
+                // print client receipt for pinpad transaction after fiscal note
+                if (payWithCard)
+                    (_, _) = Request(CommandToPinpad, "15\t");
```

Comment:
Could be a constant, e.g. `OptionPinpadPrintReceipt`

Maybe not ignore the possible error? Is it critical if this print fails?

Reply:
This print is outside of fiscal receipt transaction(after receip is closed) and nothing wrong will happen if it fail. Status of execution of this command must not change status of whole fiscal transaction.